packages/shared/src/components/cards/common/PostCardHeader.tsx

```diff
@@ -100,8 +101,8 @@ export const PostCardHeader = ({
                 className="mr-2"
                 variant={ButtonVariant.Primary}
                 href={articleLink}
-                onClick={onReadArticleClick}
                 openNewTab={openNewTab}
+                {...combinedClicks(onReadArticleClick)}
```

Comment:
Why was this change relevent to something on userStreak?

Reply:
I've seen that left click and wheel click on mouse behaved not the same (only middle click was updating the streak), so changed like to make sure not behave differently

Reply:
Doesn't it get added already via button:
https://github.com/dailydotdev/apps/blob/0147b25feae9fc51f4fcf24b3c37f8ad380d7edb/packages/shared/src/components/buttons/Button.tsx#L87

If not we missed some tracking events here potentially :O .
It should have already tracked each type of click on the read article button.

Reply:
I've tried with/without and seems to be a query issue more then a onClick one! Reverted ✅